### DISCUSSION TITLE
Tintin++ custom compilation under Cygwin

### DISCUSSION THREAD

**Discussion by CryAngel1970** [scandum/tintin#211] [General]
hello. I compiled tintin++ on Cygwin.

After compilation was completed, tt++.exe, cygwin1.dll, mintty.exe, and cygwin-console-helper.exe were copied to a new folder.

And, I created a wintin.bat file.
@ECHO OFF
%CD%\mintty.exe -w normal -c mintty.con -i tt++.ico -t WinTin++ -e ./tt++.exe -G -r main.txt

When I run wintin.bat, the mintty.exe window appears and disappears immediately.

For testing, I downloaded and installed the precompiled Windows WinTin++ 2.02.41 installer from https://tintin.mudhalla.net/download.php.

The mintty.exe window runs normally.

How do I compile my modified tintin++ source code in CygWin and run it without the window disappearing?

**Comment by scandum** [comment]
It's been over a decade since I set up WinTin++, so I honestly have no idea.

I also use an old 32 bit Cygwin version that seems to no longer be available.

WSL is preferable for modern systems, much better performance.
